packages/core/upload/admin/src/components/UploadAssetDialog/AddAssetStep/tests/FromComputerForm.test.js

```diff
@@ -425,7 +425,7 @@ describe('FromComputerForm', () => {
                       <span
                         class="c8"
                       >
-                        Drag & Drop here or
+                        Click to add an asset or drag and drop one in this area
```

Comment:
the message here should stay "Drag & Drop here or"
<img width="860" alt="image" src="https://user-images.githubusercontent.com/71838159/179547459-77a0a113-b115-4364-97b7-d25f7ef405ef.png">

Reply:
done